src/axios/axios.jsx

```diff
@@ -0,0 +1,71 @@
+const methods = {
+    GET: "GET",
+    PUT: "PUT",
+    HEAD: "HEAD",
+    POST: "POST",
+    PATCH: "PATCH",
+    DELETE: "DELETE",
+    OPTIONS: "OPTIONS",
+};
+
+const axios = function (config) {
+    const { url, method } = config;
+    switch (method) {
+        case methods.POST:
+            return axios.post(url, config);
+        case methods.DELETE:
+            return axios.delete(url, config);
+        case methods.PUT:
+            return axios.put(url, config);
+        case methods.PATCH:
+            return axios.patch(url, config);
+        case methods.HEAD:
+            return axios.head(url, config);
+        case methods.OPTIONS:
+            return axios.options(url, config);
+        case methods.GET:
+        default:
+            return axios.get(url, config);
+    }
+};
+
+axios.request = function (...args) {
+    return axios(...args);
+};
+
+axios.get = (url, config = {}) => {
```

Comment:
improve more you just wrote wrapper to fetch no difference

`axios.get('http://razorpay.com', {data:1, data2: 2}, {headers: {`content-type`: 'application/json'}})`

something like this

make data as second param, third one is config

by default take header content-type: application/json

if it's `GET` request convert the second param (object) to query param 
if `POST` request then pass data(2nd param) in body